tests/foreman/installer/test_installer.py

```diff
@@ -1390,6 +1393,45 @@ def test_installer_options_and_sections(filter):
     assert previous == current, msg
 
 
+@pytest.mark.e2e
+@pytest.mark.tier1
+@pytest.mark.parametrize("sat_ready_rhel", [settings.server.version.rhel_version], indirect=True)
+def test_satellite_installation(sat_ready_rhel):
+    """Run a basic Satellite installation
+
+    :id: bbab30a6-6861-494f-96dd-23b883c2c906
+
+    :steps:
+        1. Get a RHEL host
+        2. Configure repos
+        3. Enable satellite module
+        4. Install satellite
+        5. Run satellite-installer
+
+    :expectedresults:
+        1. Correct satellite packaged is installed
+        2. satellite-installer runs successfully
+        3. Health check runs successfully
+
+    :CaseImportance: High
+    """
+    sat_version = settings.server.version.release
+    # Register for RHEL8 repos, get Ohsnap repofile, and enable and download satellite
+    sat_ready_rhel.register_to_cdn()
+    sat_ready_rhel.download_repofile(product='satellite', release=settings.server.version.release)
+    sat_ready_rhel.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
+    installed_version = sat_ready_rhel.execute('rpm -qa satellite').stdout
+    assert sat_version in installed_version
+    # Install Satellite
+    result = sat_ready_rhel.execute(
+        InstallerCommand('scenario satellite').get_command(), timeout='30m'
+    )
+    assert result.status == 0
```

Comment:
This line has changed locally since we are now checking that there were no `ERROR` in the installer logs.

To answer your question, I believe you could do that. When an assertion fails in pytest it will show you what the variable was evaluated as, so if you had 

```python
assert 'FAIL' not in result.stdout
```
it would fail and show you what result.stdout returned in the output.